### PR TITLE
Add template with navbar & update app.py

### PR DESCRIPTION
**Pull request by Yarinh55**
closing issue #27

### CODE REVIEW

**Comment by Yarinh55** [comment]
@lioramilbaum 
Added route for subscribe

File: Vagrantfile [hunk 1]
```diff
@@ -4,3 +4,5 @@ Vagrant.configure("2") do |config|
   path:"scripts/bootstrap.sh"
   config.vm.network "forwarded_port", guest: 5000, host: 5000 
 end
+
```
Comment (lmilbaum) [inline Vagrantfile]: Remove those blank lines

File: app.py [hunk 2]
```diff
@@ -1,12 +1,21 @@
...
 
 app = Flask(__name__)
 
 
-@app.route('/')
+''''@app.route('/')
 def hello_world():
-    return 'Hello World'
+   return 'Hello World'
+'''
```
Comment (lmilbaum) [inline app.py]: Remove commented lines if not required

File: ProvisionScripts.sh [hunk 3]
```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+echo "Hello world"
```
Comment (lmilbaum) [inline ProvisionScripts.sh]: How this file related to this issue/PR?

File: app.py [hunk 4]
```diff
@@ -1,12 +1,21 @@
...
+
+@app.route('/')
+def Home():
+    return render_template('Home.html')
+	
+@app.route('/')
+def Subscribe():
+    return render_template('#')
```
Comment (lmilbaum) [inline app.py:12]: Remove it. It is not part of this PR.  The instructions I provide you were wrong :-)

**Review by lmilbaum — CHANGES_REQUESTED** [review]
I haven't tested yet this PR. Please address all comments and I will test it after.
Keen in mind that the number of commits should be one or two at most. squash them if required.

File: ProvisionScripts.sh (hunk 3 above)
Comment (Yarinh55) [reply ProvisionScripts.sh]: removed

File: app.py (hunk 4 above)
Comment (Yarinh55) [reply app.py:12]: I put it to see navbar with 2 categories at least. we can update the rout later when the page will be implement. what do tou think?

File: Vagrantfile (hunk 1 above)
Comment (Yarinh55) [reply Vagrantfile]: done

File: app.py (hunk 2 above)
Comment (Yarinh55) [reply app.py]: done
